input/tar2/goconvey.goconvey

```diff
@@ -0,0 +1,2 @@
+#-timeout=30s
+-short
```

Comment:
Oh, there it is: `testutil.Convey_IfSlowTests` is a thing now.

This is super ugly, I know, and easy to miss: it's one layer nested inside of the IfHaveRoot check.

I want a system for specifying multiple requirements sanely; this ain't it.  :soon: